components/global/header/ProfileButton.tsx

```diff
@@ -65,28 +68,30 @@ function ProfileButton() {
                 </Button>
               </div>
             </div>
-            <hr className="border-foreground mx-auto w-full opacity-10 md:w-full" />
-            <div className="bg-background-overlay py-space-y flex min-h-[300px] flex-col items-center justify-center rounded-sm">
-              <div className="mb-4">
-                <Icon name="adgangsplatformen" />
-              </div>
-              <SheetDescription className="text-typo-heading-4 text-foreground">
-                Login via Biblotekernes
-              </SheetDescription>
-              <SheetDescription className="text-typo-heading-4 text-foreground mb-4">
-                fælles loginside
-              </SheetDescription>
-              <div>
-                <Button
-                  theme="primary"
-                  onClick={() =>
-                    router.push(`https://dapple-cms.docker/login?current-path=/go-login`)
-                  }
-                  ariaLabel="Log ind med Adgangsplatformen">
-                  LOG IND
-                </Button>
-              </div>
-            </div>
+            {adgangsplatformenLoginUrl && (
+              <>
+                <hr className="border-foreground mx-auto w-full opacity-10 md:w-full" />
+                <div className="bg-background-overlay py-space-y flex min-h-[300px] flex-col items-center justify-center rounded-sm">
+                  <div className="mb-4">
+                    <Icon name="adgangsplatformen" />
+                  </div>
+                  <SheetDescription className="text-typo-heading-4 text-foreground">
+                    Login via Biblotekernes
+                  </SheetDescription>
+                  <SheetDescription className="text-typo-heading-4 text-foreground mb-4">
+                    fælles loginside
+                  </SheetDescription>
```

Comment:
Consider these lines of text in the same SheetDescription wrapper

```suggestion
                  <SheetDescription className="text-typo-heading-4 text-foreground">
                    Login via Biblotekernes fælles loginside
                  </SheetDescription>
```

Reply:
If I do like you suggest it looks like this:

![image](https://github.com/user-attachments/assets/638a7071-9c28-458f-9c8f-c3c95ab4f521)

Thats why I ended up using two SheetDescription's